test/e2e/util/utils.go

```diff
@@ -737,7 +767,7 @@ func CreateExclusionList(namespace string, list pkgUtil.ExclusionList) env.Func
 		excluded := corev1.ConfigMap{
 			ObjectMeta: metav1.ObjectMeta{
 				GenerateName: "excluded",
-				Namespace:    EraserNamespace,
+				Namespace:    TestNamespace,
```

Comment:
This function accepts a `namespace` parameter, so this should be replaced with `namespace`